backend/controller/admin/local_cmd_client.go

```diff
@@ -0,0 +1,59 @@
+package admin
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/common/configuration"
+)
+
+// localCmdClient reads and writes to local projectconfig files without making any network
+// calls. It allows us to interface with local ftl-project.toml files without needing to
+// start a controller.
+type localCmdClient struct {
+	as *AdminService
```

Comment:
If you just embed this it will expose all of the functions for the interface automatically, and you won't need to wrap them below at all.

Reply:
oo la la!